### ISSUE TITLE
Watch only *.js?

### ISSUE THREAD

**Issue by k33g** [mklabs/tilt#6]
- If I update a `js` file, application is reloaded
- If I update a `jsx` file, application isn't reloaded

**Comment by mklabs** [comment]
Yep.

What should we do ? This is a bit trickier because of babel-register.

Maybe something similar to https://github.com/reactjs/express-react-views/blob/master/index.js#L58

**Comment by mklabs** [comment]
Fixed it with https://github.com/mklabs/tilt-react-views/commit/f9a673f1202ff7efa1ff8b323c691e1a3f9771c9

You should update tilt-react-views to get the fix. `rm -rf node_modules/tilt-react-views && npm install`

**Comment by k33g** [comment]
seems not work

**Comment by mklabs** [comment]
it worked for me.

Can you confirm that `node_modules/tilt-react-view/index.js` is the same as https://github.com/mklabs/tilt-react-views/blob/master/index.js#L58 ?

**Comment by mklabs** [comment]
The application won't reload when you change the JSX files, it'll just enable the recompilation and invalid babel cache on next request.

**Comment by k33g** [comment]
Yes I confirm that node_modules/tilt-react-view/index.js is the same as https://github.com/mklabs/tilt-react-views/blob/master/index.js#L58 ?

**Comment by mklabs** [comment]
Not sure what's going on.

Just tested, it works for me.
1. Load http://localhost:3000, see Hello Title
2. Update `examples/vertical/app/views/index.jsx` to be `Bonjour {this.props.name}`
3. Reload http://localhost:3000, see Bonjour Title

**Comment by k33g** [comment]
What I mean:  if I update a js file, nodemon live reload the web application, but it doesn't happen with jsx
but if I manually refresh, it's ok

**Comment by mklabs** [comment]
Yes, that's what I tried to explain with: 

> The application won't reload when you change the JSX files, it'll just enable the recompilation and invalid babel cache on next request.

I'm not sure it's doable with nodemon
